### ISSUE TITLE
Is the cross icon implemented?

### ISSUE THREAD

**Issue by introlinux** [AstraLuma/cropper#19]
I guess the cross icon will to move the layers, but I'm really not sure. Now it does exactly the same as the plus sign icon. Is the cross icon implemented?

**Comment by AstraLuma** [comment]
Wait, holy cow! Someone uses this?

**Comment by AstraLuma** [comment]
I honestly don't remember, I'll have to get back to you.

Many of the tools exposed in the UI aren't implemented. In addition, last I knew, there was a pretty serious zooming bug.

**Comment by introlinux** [comment]
Yes we use it. It is very useful at school to cut photos of our pupils in a scanned sheet with all of them together :-)

**Comment by introlinux** [comment]
I have not realized the zooming bug, how to get it?

**Comment by AstraLuma** [comment]
Zoom out. The crop boxes no longer line up where you drew them.
